sensor_harm/sentinel2.py

```diff
@@ -90,7 +92,7 @@ def sentinel_harmonize_sr(s2_entry, target_dir, apply_bandpass=True):
 
     bands = ['sr_band2', 'sr_band3', 'sr_band4', 'sr_band8', 'sr_band8a', 'sr_band11', 'sr_band12']
 
-    process_NBAR(s2_entry, scene_id, bands, sz_path, sa_path, vz_path, va_path, satsen, target_dir, apply_bandpass)
+    process_NBAR(s2_entry, scene_id, bands, sz_path, sa_path, vz_path, va_path, satsen, target_dir, apply_bandpass, nodata=-9999)
```

Comment:
Where nodata `-9999` from?

Reply:
When Sentinel-2 is processed using LaSRC, otherwise it is 0 and goes through the function sentinel_harmonize_SAFE using nodata=0